modules/nextflow/src/main/resources/nextflow/executor/command-run.txt

```diff
@@ -156,6 +156,13 @@ nxf_main() {
     wait $pid || nxf_main_ret=$?
     {{unstage_cmd}}
     {{after_script}}
+
+    if [ -n "$LATCH_LOG_DIR" ]; then
+        workdir_base="$(basename "$(dirname "$NXF_TASK_WORKDIR")")/$(basename "$NXF_TASK_WORKDIR")"
```

Comment:
one concern is how do you associate process execution w/ the workdir hash its under? might make sense to have a link in the ui from process node -> workdir on latch